releasenotes/notes/pin-rocky9-baseimage-3d224df5b6f56b10.yaml

```diff
@@ -0,0 +1,6 @@
+fixes:
+  - |
+    Pin the OCI image tag used for the base-image of Rocky 9
+    Kolla image builds. This prevents packages in the image with
```

Comment:
I wouldn't say prevents, but meh.

Reply:
yeah, however looks like RHEL is not updating their minor versions tags after release:
https://quay.io/repository/rockylinux/rockylinux?tab=tags&tag=latest